agents/exec/src/main/resources/org.eclipse.che.exec.script.sh

```diff
@@ -23,7 +32,7 @@ if [ ${CURL_INSTALLED} = false ] && [ ${WGET_INSTALLED} = false ]; then
   CURL_INSTALLED=true
 fi
 
-test "$(id -u)" = 0 || SUDO="sudo -E"
+is_current_user_root && is_current_user_sudoer || SUDO="sudo -E"
```

Comment:
Yes, it is what I had expected. In your example return value of function `is_current_user_sudoer` should return 1 since sudo is not installed. Then 
```
is_current_user_root && is_current_user_sudoer || SUDO="sudo -E"
echo SUDO=${SUDO}
```
echos SUDO=sudo -E

Now the logic of `set_sudo_command` function looks good.